### ISSUE TITLE
Feature Request: Dash Unit tests

### ISSUE THREAD

**Issue by scott-huberty** [lina-usc/pylossless#46]
Making this a separate issue bc Dash unit tests will require a lot of development.

We need to integrate some basic unit tests for our QC dashboard into our CI pipeline.  Some basic things like testing that there is a `layout.shape` where we know there should be a `bad_pylossless` `mne.annotation` (using a test file). 

This documenation is a good place to start: https://dash.plotly.com/testing

Adding this to the 0.01 milestone.

**Comment by christian-oreilly** [comment]
FIrst Dash test was integrated in PR #117. We need to clarify the specific deliverables or objectives that we want to meet before we can close this issue. @scott-huberty
